ntbs-service/wwwroot/css/header.scss

```diff
@@ -18,13 +18,59 @@
   } 
 }
 
-
 .header-navigation-list {
     margin-bottom: 0;
 
     .header-navigation-list-item {
         font-size: large;
     }
+
+    ul {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+    }
+  
+    ul li {
+        margin: 0;
+        padding: 5px;
+    }
+  
+    li.dropdown-menu {
+        display: inline-block;
+    }
+  
+    .dropdown-menu-content {
+        display: none;
+        position: absolute;
+        background-color: #f9f9f9;
+        box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+        z-index: 1;
+        margin-top: 13px;
+        margin-left: -13px;
+        border: 1px solid black;
+    }
+  
+    .dropdown-menu-content li {
+        width: 100%;
+        outline: 1px solid black;
+    }
+  
+    .dropdown-menu-content a {
+        color: black;
+        padding: 6px 8px;
+        text-decoration: none;
+        display: block;
+        text-align: left;
+        font-weight: 200;
+    }
+  
+    .dropdown-menu-content a:hover {background-color: #f1f1f1}
+  
+    .dropdown-menu:hover .dropdown-menu-content {
+        display: block;
+    }

```

Comment:
This definitely doesn't want to be a hover action - it means there's no easy way for keyboard users (e.g.) to get there, so it's generally an accessibilty no-no. We want to make sure it's a click-based one, and that the usual keyboard nav is respected (enter, spacebar, tabbing throught the options etc)